### PR TITLE
Other fixes for docs referencing old status bar

### PR DESCRIPTION
**Pull request by sinemakinci1**
These are updates to the docs elsewhere that reference the old status bar experience. Lower priority to our docs on troubleshooting and configuring

### CODE REVIEW

File: docs/build.md [hunk 1]
```diff
@@ -18,17 +18,17 @@ The results of the build are written to the directory specified by [cmake.buildD
...
 
 Selecting this button shows a quick pick list for all the targets CMake Tools is aware of that can be built, along with the full path to the build result that will be generated by the target:
 
-![List of targets](images/target_selector.png)
+![List of targets](images/new-cmake-target-options.png)
 
 ## Build a single target
 
-You can build a single target without changing the current build target from the VS Code command pallette by running the **CMake: Build a target** command, or by pressing the keyboard shortcut (default is `Shift+F7`). CMake will build any dependent targets, even if they aren't directly selected.
+You can build a single target without changing the current build target from the VS Code command pallette by running the **CMake: Buildtar get** command, or by pressing the keyboard shortcut (default is `Shift+F7`). CMake will build any dependent targets, even if they aren't directly selected.
```
Comment (snehara99) [inline docs/build.md:31]:
```suggestion
You can build a single target without changing the current build target from the VS Code command pallette by running the **CMake: Build Target** command, or by pressing the keyboard shortcut (default is `Shift+F7`). CMake will build any dependent targets, even if they aren't directly selected.
```

File: docs/cmake-presets.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ You can configure and build your CMake project by using a series of commands. Op
 
 ### CMake: Select Configure Preset
 
-**CMake: Select Configure Preset** lists the union of non-hidden Configure Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. Select a Configure Preset to make it the active Configure Preset. This is the `configurePreset` value that's used when CMake is invoked to generate the project build system. The active Configure Preset appears on the status bar.
+**CMake: Select Configure Preset** lists the union of non-hidden Configure Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. Select a Configure Preset to make it the active Configure Preset. This is the `configurePreset` value that's used when CMake is invoked to generate the project build system. The active Configure Preset appears on the CMake Tools side bar.
```
Comment (snehara99) [inline docs/cmake-presets.md:38]:
```suggestion
**CMake: Select Configure Preset** lists the union of non-hidden Configure Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. Select a Configure Preset to make it the active Configure Preset. This is the `configurePreset` value that's used when CMake is invoked to generate the project build system. The active Configure Preset appears on the CMake Tools sidebar.
```

File: docs/cmake-presets.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ To configure the project, run **CMake: Configure** from the command palette. Thi
 
 ### CMake: Select Build Preset
 
-**CMake: Select Build Preset** lists the Default Build Preset and the union of non-hidden Build Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The Default Build Preset is equivalent to passing `cmake --build` with no additional arguments from the command line. Select a Build Preset to make it the active Build Preset. This is the `buildPreset` value that's used when CMake is invoked to build the project. The active Build Preset is displayed on the status bar.
+**CMake: Select Build Preset** lists the Default Build Preset and the union of non-hidden Build Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The Default Build Preset is equivalent to passing `cmake --build` with no additional arguments from the command line. Select a Build Preset to make it the active Build Preset. This is the `buildPreset` value that's used when CMake is invoked to build the project. The active Build Preset is displayed on the CMake Tools side bar.
```
Comment (snehara99) [inline docs/cmake-presets.md:51]:
```suggestion
**CMake: Select Build Preset** lists the Default Build Preset and the union of non-hidden Build Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The Default Build Preset is equivalent to passing `cmake --build` with no additional arguments from the command line. Select a Build Preset to make it the active Build Preset. This is the `buildPreset` value that's used when CMake is invoked to build the project. The active Build Preset is displayed on the CMake Tools sidebar.
```

File: docs/cmake-presets.md [hunk 4]
```diff
@@ -66,21 +66,21 @@ CTest is the CMake test driver program and is integrated with the CMake Tools ex
 
 **CMake: Select Test Preset** lists the default Test Preset and the union of non-hidden Test Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The default Test Preset is the same as invoking `ctest` with no other arguments from the command line.
 
-Select a Test Preset to make it the active Test Preset. This is the `testPreset` value that will be used when you invoke CTest to run tests. The active Test Preset appears on the status bar.
+Select a Test Preset to make it the active Test Preset. This is the `testPreset` value that will be used when you invoke CTest to run tests. The active Test Preset appears on the CMake Tools side bar.
```
Comment (snehara99) [inline docs/cmake-presets.md:69]:
```suggestion
Select a Test Preset to make it the active Test Preset. This is the `testPreset` value that will be used when you invoke CTest to run tests. The active Test Preset appears on the CMake Tools sidebar in the Project Status View.
```

File: docs/cmake-presets.md [hunk 5]
```diff
@@ -66,21 +66,21 @@ CTest is the CMake test driver program and is integrated with the CMake Tools ex
...
 
 All Test Presets must specify an associated `configurePreset` value. CMake Tools will hide Test Presets that don't apply to the active Configure Preset. For more information, see the [list of Test Presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#test-preset).
 
 ### CMake: Run Tests
 
 To invoke CTest, run **CMake: Run Tests from the command palette**. This is the same as running `ctest --preset <testPreset>` from the command line, where `<testPreset>` is the name of the active Test Preset.
 
-### Status bar behavior
+### CMake Tools side bar behavior
```
Comment (snehara99) [inline docs/cmake-presets.md:77]:
```suggestion
### CMake Tools Project Status View behavior
```

Comment (snehara99) [reply docs/cmake-presets.md:77]: seems like the project outline isn't included in this section so I thought it should be more specific? up to you :)

File: docs/cmake-presets.md [hunk 6]
```diff
@@ -66,21 +66,21 @@ CTest is the CMake test driver program and is integrated with the CMake Tools ex
...
 
-The status bar displays the active Configure Preset, Build Preset, and Test Preset:
+The CMake Tools side bar displays the active Configure Preset, Build Preset, and Test Preset under their respective Configure, Build, and Test Nodes. You can select each option to change the active preset:
```
Comment (snehara99) [inline docs/cmake-presets.md:79]:
```suggestion
The CMake Tools Project Status View in the sidebar displays the active Configure Preset, Build Preset, and Test Preset under their respective Configure, Build, and Test Nodes. You can select each option to change the active preset:
```

File: docs/cmake-presets.md [hunk 7]
```diff
@@ -66,21 +66,21 @@ CTest is the CMake test driver program and is integrated with the CMake Tools ex
...
 
-![Screenshot of the Visual Studio Code status bar, showing the active Configure Preset, Build Preset, and Test Preset.](images/status_bar_preset_status.png)
+![Screenshot of the CMake side bar, showing the active Configure Preset, Build Preset, and Test Preset.](images/cmake-presets.png)
```
Comment (snehara99) [inline docs/cmake-presets.md:81]:
```suggestion
![Screenshot of the CMake sidebar, showing the active Configure Preset, Build Preset, and Test Preset.](images/cmake-presets.png)
```

File: docs/cmake-presets.md [hunk 8]
```diff
@@ -66,21 +66,21 @@ CTest is the CMake test driver program and is integrated with the CMake Tools ex
...
 
-To show or hide individual status bar icons, you can modify `cmake.statusbar.advanced` in `settings.json`.
+To show or hide individual status bar icons, you can modify `cmake.statusbar.advanced` in `settings.json`. For more information on how to configure your view, please see our [CMake view configuration docs](cmake-options-configuration.md)
```
Comment (snehara99) [inline docs/cmake-presets.md:83]:
```suggestion
To show or hide individual status bar icons, you can modify `cmake.options.advanced` in `settings.json`. For more information on how to configure your view, please see our [CMake view configuration docs](cmake-options-configuration.md)
```

File: docs/debug-launch.md [hunk 9]
```diff
@@ -15,22 +15,26 @@ By default, launching or debugging an executable target causes it to be built fi
 
 The first time you run target debugging, CMake Tools asks for you to specify a target, which will be persisted between sessions.
 
-The active launch target is shown in the status bar to the right of the *Debug* and *Play* buttons:
+The active launch target is shown in the CMake Tools side bar under the `Launch` node:
```
Comment (snehara99) [inline docs/debug-launch.md:18]:
```suggestion
The active launch target is shown in the CMake Tools sidebar Project Status View under the `Launch` node:
```

File: docs/debug-launch.md [hunk 10]
```diff
@@ -15,22 +15,26 @@ By default, launching or debugging an executable target causes it to be built fi
...
+
 ## Debugging without a launch.json
 
 CMake Tools lets you start a debugger on a target without creating a `launch.json`.
 
 > **Note:**
 > Only the debugger from Microsoft's `vscode-ms-vscode.cpptools` extension supports quick-debugging. See [Debug using a launch.json file](#debug-using-a-launchjson-file), below, for information about `launch.json` and using other debuggers.
 
-Start a debugging session on the active target by running the  *CMake: Debug* command from the VS Code command pallette, by selecting the Debug button in the status bar, or by pressing the keyboard shortcut (the default is **Ctrl+F5**).
+Start a debugging session on the active target by running the  *CMake: Debug* command from the VS Code command pallette, by selecting the Debug button in the status bar or CMake Tools side bar, or by pressing the keyboard shortcut (the default is **Ctrl+F5**).
```
Comment (snehara99) [inline docs/debug-launch.md:35]:
```suggestion
Start a debugging session on the active target by running the  *CMake: Debug* command from the VS Code command palette, by selecting the Debug button in the status bar or CMake Tools sidebar Project Status View, or by pressing the keyboard shortcut (the default is **Ctrl+F5**).
```

File: docs/debug-launch.md [hunk 11]
```diff
@@ -225,9 +229,9 @@ Depending on your configuration or your settings, there may need to be additiona
 
 ## Run without debugging
 
-You can run a target without debugging it, by running the *CMake: Run without Debugging* from VS Code's command palatte, by selecting the play button in the status bar, or by pressing the keyboard shortcut (the default `Shift+F5`).
+You can run a target without debugging it, by running the *CMake: Run without Debugging* from VS Code's command palatte, by selecting the play button in the status bar or the play button to the left of the Launch node, or by pressing the keyboard shortcut (the default `Shift+F5`).
```
Comment (snehara99) [inline docs/debug-launch.md:232]:
```suggestion
You can run a target without debugging it, by running the *CMake: Run Without Debugging* from VS Code's command palette, by selecting the play button in the status bar or the play button to the left of the Launch node, or by pressing the keyboard shortcut (the default `Shift+F5`).
```

File: docs/build.md [hunk 12]
```diff
@@ -18,17 +18,17 @@ The results of the build are written to the directory specified by [cmake.buildD
...
 
 ## Create a build task
 You can also define a build task from the VS Code command pallette by running the **Tasks: Configure task** command.
```
Comment (snehara99) [inline docs/build.md:34]:
```suggestion
You can also define a build task from the VS Code command palette by running the **Tasks: Configure Task** command.
```

**Review by gcampbell-msft — APPROVED** [review]

**Pull request merged**